cmd/registry/user.go

```diff
@@ -23,8 +23,8 @@ import (
 )
 
 var usersListCmd = &cobra.Command{
-	Use:   "users",
-	Short: "List the users within an org",
+	Use:   "usersByOrg",
```

Comment:
the original name `users` is the preferred way, as it's consistent with the CRUD naming convention and used in other commands (list nodes, list services, list environments, etc.)

Reply:
Changed to `users`